test/integration/Bijectors/runtests.jl

```diff
@@ -0,0 +1,210 @@
+        if case.broken === Both || case.broken === Forward
+            @test_broken(
+                Enzyme.gradient(f_mode, Enzyme.Const(f), x...)[1] ≈ finitediff,
+                rtol = rtol,
+                atol = atol,
+            )
+        else
+            @test(
+                Enzyme.gradient(f_mode, Enzyme.Const(f), x...)[1] ≈ finitediff,
+                rtol = rtol,
+                atol = atol,
+            )
+        end
+    end
+
+    if !(case.skip === Reverse) && !(case.skip === Both)
+        if case.broken === Both || case.broken === Reverse
+            @test_broken(
+                Enzyme.gradient(r_mode, Enzyme.Const(f), x...)[1] ≈ finitediff,
+                rtol = rtol,
+                atol = atol,
+            )
+        else
+            @test(
+                Enzyme.gradient(r_mode, Enzyme.Const(f), x...)[1] ≈ finitediff,
+                rtol = rtol,
+                atol = atol,
+            )
+        end
+    end
+    return nothing
+end
+
+"""
+A helper function that returns a TestCase that evaluates sum(bijector(inverse(bijector)(x)))
+"""
+function sum_b_binv_test_case(
+    bijector, dim; runtime_activity=Neither, name=nothing, broken=Neither, skip=Neither
+)
+    if name === nothing
+        name = string(bijector)
+    end
+    b_inv = Bijectors.inverse(bijector)
+    return TestCase(
+        x -> sum(bijector(b_inv(x))),
+        randn(rng, dim);
+        runtime_activity=runtime_activity, name=name, broken=broken, skip=skip
+    )
+end
+
+@testset "Bijectors integration tests" begin
+    test_cases = TestCase[
+        sum_b_binv_test_case(Bijectors.VecCorrBijector(), 3),
+        sum_b_binv_test_case(Bijectors.VecCorrBijector(), 0),
+        sum_b_binv_test_case(Bijectors.CorrBijector(), (3, 3)),
+        # TODO(mhauru) Skip Reverse because of
+        # https://github.com/EnzymeAD/Enzyme.jl/issues/2033
+        sum_b_binv_test_case(Bijectors.CorrBijector(), (0, 0); skip=Reverse),
+        sum_b_binv_test_case(Bijectors.VecCholeskyBijector(:L), 3),
+        sum_b_binv_test_case(Bijectors.VecCholeskyBijector(:L), 0),
+        sum_b_binv_test_case(Bijectors.VecCholeskyBijector(:U), 3),
+        sum_b_binv_test_case(Bijectors.VecCholeskyBijector(:U), 0),
+        sum_b_binv_test_case(Bijectors.Coupling(Bijectors.Shift, Bijectors.PartitionMask(3, [1], [2])), 3),
+        sum_b_binv_test_case(Bijectors.InvertibleBatchNorm(3), (3, 3)),
+        sum_b_binv_test_case(Bijectors.LeakyReLU(0.2), 3),
+        sum_b_binv_test_case(Bijectors.Logit(0.1, 0.3), 3),
+        sum_b_binv_test_case(Bijectors.PDBijector(), (3, 3)),
+        sum_b_binv_test_case(Bijectors.PDVecBijector(), 3),
+        sum_b_binv_test_case(
+            Bijectors.Permute([
+                0 1 0;
+                1 0 0;
+                0 0 1
+            ]),
+            (3, 3),
+        ),
+        # TODO(mhauru) Both modes broken because of
+        # https://github.com/EnzymeAD/Enzyme.jl/issues/2035
```

Comment:
So this issue is essentially due to Enzyme 0.13 not being supported by the custom rule on bijectors on all uses of roots.jl to define rules.

x/ref: https://github.com/TuringLang/Bijectors.jl/issues/339

I think this needs to be fixed in bijectors.jl before we can take this in here

Reply:
I would propose merging this as is while we fix this, to get the vast majority of the tests in this PR into CI. Bijectors is pinned to a specific version, so once the necessary changes have been made in Bijectors we can release a new version of IT, and then make a PR that marks this unbroken as it bumps the Bijectors version of these tests.